lamindb_setup/dev/_hub_client.py

```diff
@@ -89,9 +89,13 @@ def get_access_token(email: Optional[str] = None, password: Optional[str] = None
 
 
 def lamindb_client_config_settings(settings: BaseSettings) -> Dict[str, Any]:
-    connector_file, _ = urlretrieve(
-        "https://lamin-site-assets.s3.amazonaws.com/connector.env"
-    )
+    if os.environ["LAMIN_ENV"] == "staging":
```

Comment:
This here is problematic as users won't have "LAMIN_ENV" set, and then this will error.

Reply:
You have to use `os.getenv("LAMIN_ENV")` here, instead. This won't raise an error but return `None` when an env variable doesn't exist.